Sources/Data/Model/Experiences/Row.swift

```diff
@@ -71,6 +71,27 @@ extension Row: Decodable {
                 throw DecodingError.dataCorruptedError(forKey: CodingKeys.typeName, in: container, debugDescription: "Expected one of BarcodeBlock, ButtonBlock, ImageBlock, RectangleBlock, TextBlock, WebViewBlock – found \(typeName)")
             }
         }
+        
```

Comment:
I don't think you actually need BlockType to be Encodable since its only a utility type used when decoding a heterogeneous array of blocks.